### ISSUE TITLE
How to use pre-trained weight to test custom dataset?

### ISSUE THREAD

**Issue by tingenyeh** [KaiyangZhou/deep-person-reid#264]
Hello,
Thanks for your work!

After I register my own dataset(pids=25) followed by [issue # 207](https://github.com/KaiyangZhou/deep-person-reid/issues/207),  how do I use the pre-trained weight(market1501) to test my dataset? 

the following codes :

"torchreid.data.register_image_dataset('mydataset', Mydataset)"
"model = torchreid.models.build_model(
    name='osnet_x1_0',
    num_classes=datamanager.num_train_pids,
    loss='softmax',
    pretrained=True
)"
Here we only use imagenet pre-trained weight, am I right?

How do I use the market1501 or other pre-trained weight on the model?
Thank you!!

**Comment by KaiyangZhou** [comment]
`torchreid.utils.load_pretrained_weights(model, weight_path)`

**Comment by KaiyangZhou** [comment]
@KaiyangZhou 
Add this to the how-to instructions
